Automattic/jetpack
# Protect: Fix alignment of long navigation items on mobile

Fixes an issue where multi-line navigation labels (i.e. a long plugin name) are center aligned. This PR just adds a `text-align: left;` to ensure the navigation items always have the correct text alignment.

#### Changes proposed in this Pull Request:

* Add `text-align: left;` to navigation items.

#### Other information:


#### Jetpack product discussion



1199980337313591-as-1202594951621279/f

#### Does this pull request change what data or activity we track or use?



Nope!

#### Testing instructions:






* Install a plugin with a long name, such as "[Always Use Open Graph with Jetpack](https://wordpress.org/plugins/always-use-jetpack-open-graph/)". Alternatively, use your browser's dev tools to modify an existing plugin name to be longer and wrap multiple lines during the next step.
* Open the Protect admin page and make the window width ~400px, and open the navigation.
* Confirm the text is center aligned, same as the screenshots below.
* Apply this branch.
* Confirm the text is left aligned, same as the screenshots below.

#### Screenshots

Before

<img width="332" alt="Screen Shot 2022-07-22 at 2 03 27 PM" src="https://user-images.githubusercontent.com/10933065/180518699-e2e84b92-cb66-4436-a652-8b3f6dcf585d.png">

After

<img width="341" alt="Screen Shot 2022-07-22 at 1 52 25 PM" src="https://user-images.githubusercontent.com/10933065/180518665-0fc8a421-a2c0-4680-abfb-78014581d2b8.png">
